moveit_ros/perception/depth_image_octomap_updater/src/depth_image_octomap_updater.cpp

```diff
@@ -135,14 +137,18 @@ bool DepthImageOctomapUpdater::initialize()
 void DepthImageOctomapUpdater::start()
 {
   image_transport::TransportHints hints("raw", ros::TransportHints(), nh_);
-  pub_model_depth_image_ = model_depth_transport_.advertiseCamera("model_depth", 1);
 
+  std::string prefix = "";
+  if (!ns_.empty())
+    prefix = ns_ + "/";
```

Comment:
My only concern is, we might end up with two or three slashes. For example, if the user defines these parameters:

ns = "/left_realsense/"
filtered_cloud_topic = "/camera_topic"

Reply:
Although I think that's an acceptable risk and shouldn't be too hard for a user to figure out by `rostopic list`. The amount of parsing code to handle that would be excessive IMO.

In short, I approve  :+1: